### ISSUE TITLE
Carthage 0.16 archive project get a Generic Xcode Archive in other items. it's not a iOS App Archive.

### ISSUE THREAD

**Issue by foxling** [Carthage/Carthage#1263]
I used a empty project to test. When i added New Run Script Phase to copy frameworks, and archive project. I get a archive file as 'Generic Xcode Archive' in other items. it's not a iOS App Archive.

* `carthage version`: 0.16
* `xcodebuild -version`: Xcode 7.3 Build version 7D175
* Are you using `--no-build`? No
* Are you using `--no-use-binaries`? No
* Are you using `--use-submodules`? No

**Cartfile**
```
github "Alamofire/Alamofire"
```

**Carthage Output**
```
*** Fetching Alamofire
*** Checking out Alamofire at "3.3.1"
*** xcodebuild output can be found in /var/folders/mc/wq0zkp7s1ssd997ychtxqmlw0000gn/T/carthage-xcodebuild.er5eTY.log
*** Building scheme "Alamofire tvOS" in Alamofire.xcworkspace
*** Building scheme "Alamofire OSX" in Alamofire.xcworkspace
*** Building scheme "Alamofire watchOS" in Alamofire.xcworkspace
*** Building scheme "Alamofire iOS" in Alamofire.xcworkspace
```

**Comment by renatorodrigues** [comment]
This problem has been reported in #1259 and a fix – #1261 – has already been merged into the master.

**Comment by foxling** [comment]
@renatorodrigues Sorry, i didn't see that. Thank you.
